cnxeasybake/oven.py

```diff
@@ -254,6 +257,19 @@ def bake(self, element, last_step=None):
 
         # Do label/link updates
 
+    def report_coverage_zero(self, rule):
+        """Add entry to coverage saying this selector was parsed"""
+        self.coverage_lines.append('DA:{},0'.format(rule.source_line))
+
+    def report_coverage(self, rule):
+        """Add entry to coverage saying this selector was matched"""
+        logger.debug(u'Rule ({}): {}'.format(*rule))
+        self.coverage_lines.append('DA:{},1'.format(rule[0]))
```

Comment:
since the only conditional for execution is if the rule matches, all the declarations under that rule can be considered executed, at that point, no?